### PR TITLE
LPS-162366 Upgrade Process

### PR DESCRIPTION
**Pull request by p-albuquerque**
Related issue: https://issues.liferay.com/browse/LPS-162366

### CODE REVIEW

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
It should not be sent forward yet. Depends on #1011

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_23_0/ObjectFieldSettingUpgradeProcess.java [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+						 "?)"));
+
+			ResultSet resultSet = preparedStatement1.executeQuery()) {
+
+			while (resultSet.next()) {
+				preparedStatement2.setLong(1, 0);
+
+				String uuid = _portalUUID.generate();
+
+				preparedStatement2.setString(2, uuid);
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_23_0/ObjectFieldSettingUpgradeProcess.java:69]: inline `_portalUUID.generate();`

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_23_0/ObjectFieldSettingUpgradeProcess.java [hunk 2]
```diff
@@ -0,0 +1,105 @@
...
+				String shortName = resultSet.getString("name");
+
+				if (shortName.startsWith("C_")) {
+					shortName = shortName.substring(2);
+				}
+
+				preparedStatement2.setLong(
+					9, resultSet.getLong("objectFieldId"));
+
+				preparedStatement2.setString(10, "objectDefinitionName1");
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_23_0/ObjectFieldSettingUpgradeProcess.java:93]: Use `ObjectFieldSettingConstant`

File: modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_23_0/ObjectFieldSettingUpgradeProcess.java [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+				Timestamp timestamp = new Timestamp(System.currentTimeMillis());
+
+				preparedStatement2.setTimestamp(7, timestamp);
+				preparedStatement2.setTimestamp(8, timestamp);
+
+				String shortName = resultSet.getString("name");
+
+				if (shortName.startsWith("C_")) {
+					shortName = shortName.substring(2);
+				}
```
Comment (MarinhoFeliphe) [inline modules/apps/object/object-service/src/main/java/com/liferay/object/internal/upgrade/v3_23_0/ObjectFieldSettingUpgradeProcess.java:88]: Define this logic more closer of its usage.

**Review by MarinhoFeliphe — CHANGES_REQUESTED** [review]

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by MarinhoFeliphe** [comment]
LGTM!! :tada:

**Comment by naraandrade** [comment]
@p-albuquerque We have some conflicts to be resolved.

**Comment by p-albuquerque** [comment]
waiting [this](https://github.com/brianchandotcom/liferay-portal/pull/123514) PR be merged

**Comment by p-albuquerque** [comment]
ci:test:sf

**Comment by p-albuquerque** [comment]
ci:test:relevant

**Comment by p-albuquerque** [comment]
ci:forward

**Pull request closed without merging**